### DISCUSSION TITLE
speed-measure-webpack-plugin和DllPlugin如何在飞冰中使用？

### DISCUSSION THREAD

**Discussion by Sharang-heng** [alibaba/ice#6189] [Q&A(2.x)]
const SpeedMeasurePlugin = require('speed-measure-webpack-plugin');

// 参考配置https://github.com/alibaba/ice/issues/3179
module.exports = ({
  onGetWebpackConfig,
}) => {
  onGetWebpackConfig((config) => {
    config
      .plugin('BundleAnalyzerPlugin')
      .use(SpeedMeasurePlugin);
  });
};

**Comment by Sharang-heng** [comment]
我这样写但是打印出来的东西好少

**Reply by ClarkXia** [reply]
具体少指什么
